doc/contribute/doc-guidelines.rst

```diff
@@ -0,0 +1,308 @@
+.. _doc_guidelines:
+
+Documentation Guidelines for the Zephyr Project
+###############################################
+
+Zephyr Project content is written using the `reStructuredText`_ markup
+language (.rst file extension) with Sphinx extensions, and processed
+using sphinx to create a formatted standalone website.  Developers can
+view this content either in its raw form as .rst markup files, or (with
+sphinx installed) they can run the makefile (or make.bat on windows) to
```

Comment:
What about something like this (random capitalization issues, plus a kind of subtle issue in that Makefiles aren't really "run"):

```
Sphinx installed) they can build the documentation using the Makefile on Unix
systems, or make.bat on Windows, to generate the HTML content. The HTML
content can then be viewed using a web browser.
```